### ISSUE TITLE
Seemingly incorrect number of projects per group is shown

### ISSUE THREAD

**Issue by pfalcon** [Linaro/squad#709]
At https://qa-reports.linaro.org/ , I see that my group is shown as:

> Paul Sokolovsky's group
Paul Sokolovsky's qa-reports group, mostly for testing.
4 projects

However, going inside it, https://qa-reports.linaro.org/pfalcon/, there're only 2 projects (and none archived).

**Comment by pfalcon** [comment]
I decided to submit this, before creating new projects in there. Let me also provide screenshots of the current state:

![Screenshot at 2020-04-29 14-11-50](https://user-images.githubusercontent.com/500451/80589757-7898f500-8a23-11ea-89cb-e2e61b8999b9.png)

---

![Screenshot at 2020-04-29 14-12-14](https://user-images.githubusercontent.com/500451/80589768-7fc00300-8a23-11ea-9acb-7ff5a0c4b608.png)

**Comment by pfalcon** [comment]
After creating 3rd project, the count on front page shows "6". With 4th, it shows "8".

**Comment by mwasilew** [comment]
@pfalcon this is weird. I can see 4 projects in your group: zephyr-net, manual-test, zephyr-jerryscript, zephyr-micropython. However main page only shows '2' next to your name. In other words I have the opposite situation. Let me check what might be going on.

**Comment by pfalcon** [comment]
> I can see 4 projects in your group: zephyr-net, manual-test, zephyr-jerryscript, zephyr-micropython. 

Yep, just created 2 more (apparently we posted comments in about the same time).

**Comment by mwasilew** [comment]
Ah, in this case my numbers look OK when I'm logged in. When I'm not logged in I'm getting the same problem.

**Comment by pfalcon** [comment]
> Ah, in this case my numbers look OK when I'm logged in. When I'm not logged in I'm getting the same problem.

For me the situation is, with current 4 actual projects in https://qa-reports.linaro.org/pfalcon/ :

* When logged in, with existing the login session (for which the original report was submitted): count of front page is 8.
* Logged out: 8
* Private browser window: 8
* Logged in again: 8

**Comment by mwasilew** [comment]
For some reason admin users get different numbers than 'ordinary' users. I'll try to figure out why.

**Comment by pfalcon** [comment]
Thanks. This is definitely a low-priority issue, just wanted to record it.

**Comment by mwasilew** [comment]
I found the issue, it's here (somewhere): https://github.com/Linaro/squad/blob/master/squad/core/models.py#L49

**Comment by mwasilew** [comment]
The issue is that GroupManager annotation adds 1 for each group member (for each project). I was able to reproduce this with my personal space. By adding 2 members I increased to counter displayed on the main page to 3. There is just 1 project in my group. Easy fix would be to remove ```| Q(members__id=user.id)``` however I don't quite understand why this was added in the first place. @chaws, @terceiro any hints?
